validation.js

```diff
@@ -19,3 +21,23 @@ form.addEventListener('submit', (event) => {
   }
   event.preventDefault();
 });
+
+function saveInfo() {
+  const userInfo = JSON.stringify({
+    username: userName.value,
+    email: email.value,
+    text:
+     text.value,
+  });
+  localStorage.setItem('userInfo', userInfo);
+}
```

Comment:
[OPTIONAl] For best practices, flexibility, and better code readability 🤓 consider using an arrow function here, it would be something like this 👇🏽 
```
const saveInfo = () => {
  const userInfo = JSON.stringify({
    username: userName.value,
    email: email.value,
    text:
     text.value,
  });
  localStorage.setItem('userInfo', userInfo);
};
```